src/util/postgres_client.js

```diff
@@ -396,9 +402,11 @@ class BulkOp {
         let nMatched = 0;
         let nModified = 0;
         let nRemoved = 0;
+        let should_rollback = false;
         try {
             await this.transaction.begin();
             const batch_query = this.queries.join('; ');
+            should_rollback = true;
```

Comment:
Wondering about this change,
this.transaction.begin() includes -
1. initialization of this.pg_client 
2. running the first query of the transaction - BEGIN TRANSACTION

if something failed for example in this.queries.join, shouldn't we call rollback?
I know nothing happened but indeed a transaction started in the db, do we need it as a cleanup?

Reply:
@Danny with this change we still have the `BEGIN TRANSACTION` in our DB and we will not rollback it - from what I understand that @romayalon wrote.

Reply:
@shirady yes but after reading a bit I think that rollback won't clean the started transaction and actually we can keep committing updates to the transaction after rollbacking, so I guess this is okay.